### PR TITLE
Addition of new scopes for the Implement OAuth for Okta Guide

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Addition of new scopes (`okta.userTypes.manage`, `okta.userTypes.read`, `okta.profileMappings.manage`, and `okta.profileMappings.read`) to the [Implement OAuth for Okta](https://developer.okta.com/docs/guides/implement-oauth-for-okta/main/) guide.
- **Is this PR related to a Monolith release?** Yes 2021.12.0

### Resolves:

* [OKTA-449724](https://oktainc.atlassian.net/browse/OKTA-449724)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md [hunk 1]
```diff
@@ -160,7 +160,9 @@ The following table shows the scopes that are currently available:
 | `okta.linkedObjects.manage`| Allows the app to manage Linked Object definitions in your Okta organization.| [Linked Objects API](/docs/reference/api/linked-objects/)|
 | `okta.linkedObjects.read` | Allows the app to read Linked Object definitions in your Okta organization.| [Linked Objects API](/docs/reference/api/linked-objects/)|
 | `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
-| `okta.roles.manage`      | Allows the app to create and manage Administrator Roles in your Okta organization     | [Administrator Roles API](/docs/reference/api/roles/#get-started)|
+| `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
```
Comment (talbenari-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md:163]: I think you may have deleted a row from the table and re-added the wrong row back to the document

**Review by talbenari-okta — CHANGES_REQUESTED** [review]
Here are the exact descriptions provided in the UI:

- `okta.profileMappings.manage`: `Allows the app to manage user profile mappings in your Okta organization.`
- `okta.profileMappings.read`: `Allows the app to read user profile mappings in your Okta organization.`
- `okta.userTypes.manage`: `Allows the app to manage user types in your Okta organization.`
- `okta.userTypes.read`: `Allows the app to read user types in your Okta organization.`

It would probably be best to keep these descriptions consistent.

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md]: Great catch. I did. Thanks very much.

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md [hunk 2]
```diff
@@ -149,33 +149,37 @@ The following table shows the scopes that are currently available:
...
+| `okta.policies.read`      | Allows the app to read information about Policies in your Okta organization | [Policy API](/docs/reference/api/policy/#policy-api-operations)|
+| `okta.profileMappings.read`| Allows the app to read user profile mappings in your Okta organization | [Mappings API](//docs/reference/api/mappings/)|
+| `okta.profileMappings.manage`| Allows the app to manage user profile mappings in your Okta organization | [Mappings API](/docs/reference/api/mappings/)|
 | `okta.roles.read`        | Allows the app to read information about Administrator Roles in your Okta organization | [Administrator Roles API](/docs/reference/api/roles/#get-started)|
-| `okta.schemas.manage`    | Allows the app to create and manage Schemas in your Okta organization   | [Schemas API](/docs/reference/api/schemas/#getting-started)|
-| `okta.schemas.read`      | Allows the app to read information about Schemas in your Okta organization| [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.roles.manage`        | Allows the app to read information about Administrator Roles in your Okta organization | [Administrator Roles API](/docs/reference/api/roles/#get-started)|
```
Comment (talbenari-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md:168]: Alphabetizing the scopes is a great call, but minor nit - `okta.roles.manage` should be on the line above `okta.roles.read`

**Review by talbenari-okta — CHANGES_REQUESTED** [review]

**Review by talbenari-okta — APPROVED** [review]
LGTM 👍

**Pull request merged**